<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>755</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>755</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	
	
	font: 25 9pt &quot;Alibaba PuHuiTi 2.0&quot;;


	color: rgb(118, 119, 121);
}

#label_2
{
	
	font: 25 9pt &quot;Alibaba PuHuiTi 2.0&quot;;
	color:rgb(255, 255, 255);
}

#label_3
{
	color:rgb(17, 168, 255);
	font:14px;
}


QPushButton
{	
	font: 8pt &quot;Arial&quot;;
}
QPushButton:hover
{	
	color:rgb(255, 255, 255);
}
QPushButton:pressed
{
    background-color:rgba(255, 255, 255, 0)
}

#Login
{
	background-color:rgb(31, 33, 38);
}

#lineEdit_account,#lineEdit_password
{
	background-color: rgb(31, 33, 38);
	font: 25 12pt &quot;Alibaba PuHuiTi 2.0&quot;;
	color:rgb(255, 255, 255);
	border-style:none;
	padding-left:10px;
	border-radius:3px
}

#lineEdit_account:hover,#lineEdit_password:hover{
	background-color: rgb(56, 59, 67);
}
#pushButton_forgetPassword:hover,#pushButton_register:hover{
	background-color: rgb(255, 255, 255);
}


QCheckBox::indicator:unchecked{	
		background-color:rgb(49, 52, 59);
}
QCheckBox::indicator:unchecked:hover{
	background-color: rgb(56, 59, 67);
}












</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>351</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/res/Login/head.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_min">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_min{

	
	
	image: url(:/res/Login/min2.0.png);
}
QPushButton#pushButton_min:hover{

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_close
{
	image: url(:/res/Login/close2.0.png);
}
QPushButton#pushButton_close:hover
{
	
}
outline:nonoe;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>481</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Alibaba PuHuiTi 2.0</family>
      <pointsize>20</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 20pt &quot;Alibaba PuHuiTi 2.0&quot;;
color: #4DA9FF;
</string>
    </property>
    <property name="text">
     <string>账号密码登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_account">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Alibaba PuHuiTi 2.0</family>
      <pointsize>12</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;Alibaba PuHuiTi 2.0&quot;;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>请输入用户名</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_account">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <zorder>comboBox_account</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_password</zorder>
   <zorder>lineEdit_account</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton_land">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>371</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_land{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(6, 191, 255), stop:1 rgb(44, 114, 255) );
	color: rgb(255, 255, 255);
	border-radius:3px;		
	font: 75 14pt &quot;微软雅黑&quot;;
}
QPushButton#pushButton_land:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;
	font: 75 14pt &quot;微软雅黑&quot;;
}

QPushButton#pushButton_land:pressed{
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0  rgb(20, 204, 255), stop:0.5 rgb(5, 191, 255), stop:1 rgb(11, 141, 255) );
	border-style:none;
	border-radius:3px;	
	font: 75 14pt &quot;微软雅黑&quot;;
}

</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_register">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Alibaba PuHuiTi 2.0&quot;;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>249</y>
     <width>371</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="checkBox_autoLand">
      <property name="minimumSize">
       <size>
        <width>119</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Alibaba PuHuiTi 2.0</family>
        <pointsize>12</pointsize>
        <weight>3</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 12pt &quot;Alibaba PuHuiTi 2.0&quot;;</string>
      </property>
      <property name="text">
       <string>自动登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_savePassword">
      <property name="font">
       <font>
        <family>Alibaba PuHuiTi 2.0</family>
        <pointsize>12</pointsize>
        <weight>3</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 25 12pt &quot;Alibaba PuHuiTi 2.0&quot;;</string>
      </property>
      <property name="text">
       <string>保存密码</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/res/Login/Vector.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi 2.0</family>
     <pointsize>20</pointsize>
     <weight>3</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 20pt &quot;Alibaba PuHuiTi 2.0&quot;;</string>
   </property>
   <property name="text">
    <string>脑电数据管理系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
